Gruntfile.js

```diff
@@ -353,12 +359,6 @@ module.exports = function(grunt) {
   }
 
   if (XUNIT) {
-    var eslint = grunt.config('eslint');
-    eslint.options = eslint.options || {};
-    eslint.options.format = 'junit';
-    eslint.options.outputFile = 'reports/eslint.xml';
-    grunt.config('eslint', eslint);
-
     var jscs = grunt.config('jscs');
     jscs.options.reporter = 'junit';
     jscs.options.reporterOutput = 'reports/jscs.xml';
```

Comment:
Should we remove `jscs` at some point? I believe the main project is deprecated and they've merged with eslint.